dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -227,6 +227,27 @@
 
       <extract-pid file="@{es.pidfile}" property="integ.pid"/>
       <echo>Shutting down external node PID ${integ.pid}</echo>
+      <!-- verify with jps that this actually is the correct pid.
+      See if we can find the line "pid org.elasticsearch.bootstrap.Elasticsearch" in the output of jps -l.-->
+      <exec executable="jps">
+        <arg value="-l"/>
+        <redirector outputproperty="jps.pid.line">
```

Comment:
for extra safety, you can declare this one as local before the exec statement (otherwise its global). In some cases maven propagates properties from ant back to itself, so in general the logic has been paranoid here to avoid something bad happening.